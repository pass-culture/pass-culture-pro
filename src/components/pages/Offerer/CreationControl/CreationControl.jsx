import { recursiveMap, SubmitButton } from 'pass-culture-shared'
import React from 'react'
import { NavLink } from 'react-router-dom'

const CreationControl = ({ parseFormChild }) =>
  recursiveMap(
    <div
      className="field is-grouped is-grouped-centered"
      style={{ justifyContent: 'space-between' }}
    >
      <div className="control">
        <NavLink
          className="button is-secondary is-medium"
          to="/structures"
        >
          {'Retour'}
        </NavLink>
      </div>
      <div className="control">
        <SubmitButton className="button is-primary is-medium">{'Valider'}</SubmitButton>
      </div>
    </div>,
    parseFormChild
  )

CreationControl.isParsedByForm = true

export default CreationControl
