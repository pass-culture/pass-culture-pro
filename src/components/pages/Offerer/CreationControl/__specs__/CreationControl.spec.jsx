import React from 'react'
import {shallow} from 'enzyme'
import CreationControl from "../CreationControl"

describe('src | components | pages | Offerer | CreationControl ', () => {
  let props

  beforeEach(() => {
    props = {
      parseFormChild: (a) => { return a }
    }
  })

  it('should match snapshot', () => {
    // when
    const wrapper = shallow(<CreationControl {...props} />)

    // then
    expect(wrapper).toMatchSnapshot()
  })
})
