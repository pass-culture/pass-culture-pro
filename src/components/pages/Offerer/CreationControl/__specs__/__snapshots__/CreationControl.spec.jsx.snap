// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`src | components | pages | Offerer | CreationControl  should match snapshot 1`] = `ShallowWrapper {}`;
